.dockerignore

```diff
@@ -1,10 +1,17 @@
 pom.xml
```

Comment:
Thinking out loud, wouldn't it be easier to ignore everything then adding only what's needed instead of listing each folder one per one?

Reply:
something like

```
*

!war/src/main/docker/Dockerfile
!war/src/main/docker/docker-entrypoint.sh
!war/target/plugin-health-scoring.jar
```

Reply:
Something like that yes, easier to maintain I think.